.github/workflows/meeting-reminder.yaml

```diff
@@ -48,10 +48,10 @@ jobs:
             unser nächstes Treffen findet am kommenden Montag (${{ steps.date.outputs.next_date }}) um 19:30 Uhr in den Räumen
             des Netz39 e.V. (Leibnizstr. 32) statt.
 
-            Falls ihr hybrid teilnehmen möchtet, gebt uns bitte bis 2h vorher kurz
+            Falls ihr hybrid teilnehmen möchtet, gebt uns bitte bis 2h vorher per Mail an kontakt@md.freifunk.net
             Bescheid, dann kümmern wir uns um die Online-Variante.
 
             Viele Grüße,
             Das ffmd-Team
         env:
-          APPRISE_URL: mailtos://${{ secrets.APPRISE_MAIL_USER }}:${{ secrets.APPRISE_MAIL_PASS }}@mail.netz39.de/?to=ffmd@netz39.de,ffmd-dev@netz39.de,announce@netz39.de&from=kontakt@md.freifunk.net
+          APPRISE_URL: mailtos://mail.netz39.de/?user=${{ secrets.APPRISE_MAIL_USER }}&pass=${{ secrets.APPRISE_MAIL_PASS }}&to=ffmd@netz39.de,ffmd-dev@netz39.de,announce@netz39.de
```

Comment:
Ich würde das gern zurückstellen, bis unsere Datumsauswahl funktioniert. Die Mail lässt sich wesentlich schwerer zurückrufen als die Discord-Nachricht. Vielleicht in einen eigenen PR, den wir dann nachlegen?

Reply:
/done, übrige Änderungen in https://github.com/FreifunkMD/.github/tree/fix-send-mail-job